### ISSUE TITLE
install error

### ISSUE THREAD

**Issue by binarygeek119** [newelement/digital-movie-poster#48]
on a fresh install i saw errors but i wasnt able to save them fast enought.
when running update.sh i get this. they might be the same errors.
```
Deploy script started
/var/www/html
PHP Warning:  require(/var/www/html/vendor/autoload.php): Failed to open stream: No such file or directory in /var/www/html/artisan on line 18
PHP Fatal error:  Uncaught Error: Failed opening required '/var/www/html/vendor/autoload.php' (include_path='.:/usr/share/php') in /var/www/html/artisan:18
Stack trace:
#0 {main}
  thrown in /var/www/html/artisan on line 18
```

**Comment by binarygeek119** [comment]
i was able to catch it this time

```
/var/www/html
Cloning into '.'...
remote: Enumerating objects: 11109, done.
remote: Counting objects: 100% (3061/3061), done.
remote: Compressing objects: 100% (979/979), done.
remote: Total 11109 (delta 1978), reused 2986 (delta 1925), pack-reused 8048
Receiving objects: 100% (11109/11109), 22.32 MiB | 9.83 MiB/s, done.
Resolving deltas: 100% (3740/3740), done.
Composer plugins have been disabled for safety in this non-interactive session. Set COMPOSER_ALLOW_SUPERUSER=1 if you want to allow plugins to run as root/super user.
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - tijsverkoyen/css-to-inline-styles is locked to version 2.2.5 and an update of this package was not requested.
    - tijsverkoyen/css-to-inline-styles 2.2.5 requires ext-dom * -> it is missing from your system. Install or enable PHP's dom extension.
  Problem 2
    - zanysoft/laravel-zip is locked to version 2.0.3 and an update of this package was not requested.
    - zanysoft/laravel-zip 2.0.3 requires ext-zip * -> it is missing from your system. Install or enable PHP's zip extension.
  Problem 3
    - phar-io/manifest is locked to version 2.0.3 and an update of this package was not requested.
    - phar-io/manifest 2.0.3 requires ext-dom * -> it is missing from your system. Install or enable PHP's dom extension.
  Problem 4
    - phpunit/php-code-coverage is locked to version 9.2.19 and an update of this package was not requested.
    - phpunit/php-code-coverage 9.2.19 requires ext-dom * -> it is missing from your system. Install or enable PHP's dom extension.
  Problem 5
    - phpunit/phpunit is locked to version 9.5.27 and an update of this package was not requested.
    - phpunit/phpunit 9.5.27 requires ext-dom * -> it is missing from your system. Install or enable PHP's dom extension.
  Problem 6
    - spatie/laravel-ignition is locked to version 1.6.2 and an update of this package was not requested.
    - spatie/laravel-ignition 1.6.2 requires ext-curl * -> it is missing from your system. Install or enable PHP's curl extension.
  Problem 7
    - theseer/tokenizer is locked to version 1.2.1 and an update of this package was not requested.
    - theseer/tokenizer 1.2.1 requires ext-dom * -> it is missing from your system. Install or enable PHP's dom extension.
  Problem 8
    - tijsverkoyen/css-to-inline-styles 2.2.5 requires ext-dom * -> it is missing from your system. Install or enable PHP's dom extension.
    - laravel/framework v9.43.0 requires tijsverkoyen/css-to-inline-styles ^2.2.5 -> satisfiable by tijsverkoyen/css-to-inline-styles[2.2.5].
    - laravel/framework is locked to version v9.43.0 and an update of this package was not requested.

To enable extensions, verify that they are enabled in your .ini files:
    - /etc/php/8.2/cli/php.ini
    - /etc/php/8.2/cli/conf.d/10-opcache.ini
    - /etc/php/8.2/cli/conf.d/10-pdo.ini
    - /etc/php/8.2/cli/conf.d/20-calendar.ini
    - /etc/php/8.2/cli/conf.d/20-ctype.ini
    - /etc/php/8.2/cli/conf.d/20-exif.ini
    - /etc/php/8.2/cli/conf.d/20-ffi.ini
    - /etc/php/8.2/cli/conf.d/20-fileinfo.ini
    - /etc/php/8.2/cli/conf.d/20-ftp.ini
    - /etc/php/8.2/cli/conf.d/20-gettext.ini
    - /etc/php/8.2/cli/conf.d/20-iconv.ini
    - /etc/php/8.2/cli/conf.d/20-imagick.ini
    - /etc/php/8.2/cli/conf.d/20-phar.ini
    - /etc/php/8.2/cli/conf.d/20-posix.ini
    - /etc/php/8.2/cli/conf.d/20-readline.ini
    - /etc/php/8.2/cli/conf.d/20-shmop.ini
    - /etc/php/8.2/cli/conf.d/20-sockets.ini
    - /etc/php/8.2/cli/conf.d/20-sysvmsg.ini
    - /etc/php/8.2/cli/conf.d/20-sysvsem.ini
    - /etc/php/8.2/cli/conf.d/20-sysvshm.ini
    - /etc/php/8.2/cli/conf.d/20-tokenizer.ini
You can also run `php --ini` in a terminal to see which files are used by PHP in CLI mode.
Alternatively, you can run Composer with `--ignore-platform-req=ext-dom --ignore-platform-req=ext-zip --ignore-platform-req=ext-curl` to temporarily ignore these required extensions.
```

**Comment by newelement** [comment]
I will run a fresh install locally and see what happens

**Comment by newelement** [comment]
Looks like none of the PHP extensions were installed previously. This is a problem I've run into. For some reason the PHP extensions fail to install for whatever reason. It does not make any sense.

**Comment by binarygeek119** [comment]
i believe the app runs on php 8.1 right? i see in the install 8.2 install instead. could this the the error?

**Comment by binarygeek119** [comment]
i also see this coming up in the installer.

```
ERROR: Module php7.4 does not exist!
Considering dependency mpm_prefork for php8.1:
Considering conflict mpm_event for mpm_prefork:
Considering conflict mpm_worker for mpm_prefork:
Module mpm_prefork already enabled
Considering conflict php5 for php8.1:
Module php8.1 already enabled
Module rewrite already enabled

```

**Comment by newelement** [comment]
I'm going to audit the install script soon Not sure If I will be able to get to it this week or not.

**Comment by newelement** [comment]
Install script is fixed for PHP 8.2
